src/app/organization/organization-routing.module.ts

```diff
@@ -56,6 +56,8 @@ import { EditOfficeComponent } from './offices/edit-office/edit-office.component
 import { AdhocQueryTemplateResolver } from './adhoc-query/adhoc-query-template.resolver';
 import { ViewLoanProvisioningCriteriaComponent } from './loan-provisioning-criteria/view-loan-provisioning-criteria/view-loan-provisioning-criteria.component';
 import { LoanProvisioningCriteriasResolver } from './loan-provisioning-criteria/loan-provisioning-criterias.resolver';
+import { ViewCashierComponent } from './tellers/view-cashier/view-cashier.component';
```

Comment:
why has component been placed with resolvers?

Reply:
Automatically imported. I'll place it in correct posiotion